mmedit/models/editors/stylegan3/stylegan3_modules.py

```diff
@@ -169,19 +175,21 @@ def __init__(self,
                 activation='lrelu',
                 lr_multiplier=lr_multiplier)
             setattr(self, f'fc{idx}', layer)
-        self.register_buffer('w_avg', torch.zeros([style_channels]))
+        if num_ws is not None and w_avg_beta is not None:
```

Comment:
Whether register ``w_avg`` is relevant to ``num_ws``. We may just remove this cond.